web/pandas/_templates/layout.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <script type="text/javascript">
+            var _gaq = _gaq || []; _gaq.push(['_setAccount', 'UA-27880019-2']); _gaq.push(['_trackPageview']);
+            (function() {
+                var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
+                ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
+                var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
```

Comment:
Can you use the same one as for the docs? 
https://github.com/pandas-dev/pandas/blob/372a9a03608f61e4472f2ba0f4b52956f8e1cce1/doc/source/themes/nature_with_gtoc/layout.html#L98-L106

(or is there a reason the above is better? then we can update the docs ones)

Also, can this go at the bottom scripts? Or is there a good reason to load this first?

Reply:
I copied the one from the website, didn't know the docs had a different one.

I followed Google instructions on where to place it, they say immediately after the `<head>` tag.

Reply:
I recently update the docs one, it was on my to do list to also update the website but didn't get to that yet ... (kind of proving your point here :-))